tests/inputText.spec.ts

```diff
@@ -1,28 +1,39 @@
-import { chromium, test } from "@playwright/test";
+import { chromium, expect, test } from "@playwright/test";
 
-test("Typing a joke and pun should only appear on the page after clicking the Submit Joke button", async () => {
+test("Typing a joke and a pun and clicking the seperate buttons should render the text on the homepage", async () => {
+  // Simualte a chromium browser with an attribute to slow the running time down by 3 seconds so that you can visually see the test going through each step.
   const browser = await chromium.launch({
     headless: false,
     slowMo: 3000,
   });
   const context = await browser.newContext();
   const page = await context.newPage();
+  // 1. user enters page.
+  await page.goto("http://localhost:5173");
+  // 2. user inputs text with placeholder.
   await page
-    .locator("#joke", {
-      hasText: "Why do scientists not trust atoms?",
-    })
-    .locator(".submit-joke")
-    .click({ force: true });
-  await page.locator(
-    '.showJokePun:has-text("Why do scientists not trust atoms?")'
-  );
+    .getByPlaceholder("Type something funny")
```

Comment:
This is brilliantly neat testing that is clearly understandable with all steps we have made, we have done a great job